ansible/roles/csm.rr.ceph_zoning/files/ceph_zoning.py

```diff
@@ -58,28 +57,7 @@ def run_command(command: str) -> str:
         raise ValueError(f"Command {command} errored out with : {e.stderr}") from e
     return result.stdout
 
-def get_ceph_zone_prefix() -> str:
-    """
-    Get ceph zone prefix if defined from the site-init secret (customization).
-    Return ceph_zone_prefix to be used by create_and_map_racks(positions_dict) function
-    for mapping racks to zones.
-    """
-    namespace = "loftsman"
-    secret_name = "site-init"
-    secret = run_command(f"kubectl -n {namespace} get secret {secret_name} -o json")
-    secret_data = json.loads(secret)
-    encoded_yaml = secret_data["data"]["customizations.yaml"]
-    decoded_yaml = base64.b64decode(encoded_yaml).decode("utf-8")
-    output_file = "/tmp/customizations.yaml"
-    with open(output_file, "w") as f:
-        f.write(decoded_yaml)
-    ceph_key_path = "spec.kubernetes.services.ceph_zone_prefix"
-    # Run yq command to extract the value
-    ceph_zone = run_command(f"yq r {output_file} {ceph_key_path}")
-    ceph_zone_prefix = ceph_zone.strip()
-    return ceph_zone_prefix
-
-def create_and_map_racks(positions_dict: Dict[str, List[str]]) -> int:
+def create_and_map_racks(positions_dict: Dict[str, List[str]], ceph_zone_prefix: str) -> List[int]:
```

Comment:
The function docstring should have the description of `sn_count_in_rack` updated -- currently it does not seem to describe what it actually is.